socs/agents/hwp_pmx/agent.py

```diff
@@ -260,6 +295,83 @@ def _stop_acq(self, session, params=None):
         else:
             return False, 'acq is not currently running'
 
+    def initiate_shutdown(self, session, params):
+        """
```

Comment:
These task docstrings all need the method signature override at the top, and to be marked with `**Task**` or `**Process**` appropriately. See the [example here](https://ocs.readthedocs.io/en/main/developer/agent_references/documentation.html#task-and-process-documentation).